### ISSUE TITLE
doReadTransfer etc. must not be called any more

### ISSUE THREAD

**Issue by killenb** [ChimeraTK/DeviceAccess#121]
Child of ChimeraTK/ApplicationCore#123
Depends on #117 

All code which is executing the read and write sub-steps separately must not call doXxxTransferYyy directly any more, but call xxxTransferYyy (e.g. readTransferLatest() instead of doReadTransferLatest()).

This concerns all decorator-like Accessor-Implementations, for instance (list not complete)
* LNMBackendBitAccessor
* LNMBackendChannelAccessor
* NDRegisterAccessorDecorator

The TransferGroup is done in a separate ticket (#120)

* [x] Find all places in DeviceAccess where a doXxxTransferYyy function is used and replace it with xxxTransferYyy (except for the TransferGroup)
* [x] List all the classes you found in this ticket
* [x] Create a follow-up ticket for each of them that a test for this functionality should be written

**Comment by mhier** [comment]
Does it hurt if I fix this aspect in the TransferGroup already?

**Comment by mhier** [comment]
Only found the problem in TransferGroup, all the others were already fixed in #117. Since the TransferGroup already has the follow-up ticket (#120) I think there is no follow up ticket to write.
